Sources/TunnelKitManager/NetworkExtensionVPN.swift

```diff
@@ -240,11 +240,11 @@ public class NetworkExtensionVPN: VPN {
         }
         let bundleId = connection.manager.tunnelBundleIdentifier
         log.debug("VPN status did change (\(bundleId ?? "?")): isEnabled=\(connection.manager.isEnabled), status=\(connection.status.rawValue)")
-
         var notification = Notification(name: VPNNotification.didChangeStatus)
         notification.vpnBundleIdentifier = bundleId
         notification.vpnIsEnabled = connection.manager.isEnabled
         notification.vpnStatus = connection.status.wrappedStatus
+        notification.connectionDate = connection.connectedDate
```

Comment:
I believe that this is unused.

Reply:
I suppose it is useful for connection timer or some other connection related date checks.
The connection starts as soon as the status didChange
Anyway, it is not related to RX/TX stats and I agree to remove it if you want